lib/rds/manager/Manager.rb

```diff
@@ -0,0 +1,188 @@
+require "rds/RDS"
+require "rds/loader/Loader"
+require "common/manager/Manager"
+require "rds/models/InstanceDiff"
+require "conf/Configuration"
+require "io/console"
+
+module Cumulus
+  module RDS
+    class Manager < Common::Manager
+      def resource_name
+        "RDS Database Instance"
+      end
+
+      def local_resources
+        @local_resources ||= Hash[Loader.instances.map { |local| [local.name, local] }]
+      end
+
+      def aws_resources
+        @aws_resources ||= RDS::named_instances
+      end
+
+      def unmanaged_diff(aws)
+        InstanceDiff.unmanaged(aws)
+      end
+
+      def added_diff(local)
+        InstanceDiff.added(local)
+      end
+
+      def diff_resource(local, aws)
+        puts Colors.blue("Processing #{local.name}...")
+        cumulus_version = InstanceConfig.new(local.name).populate!(aws)
+        local.diff(cumulus_version)
+      end
+
+      def migrate
+        puts Colors.blue("Will migrate #{RDS.instances.length} instances")
+
+        # Create the directories
+        rds_dir = "#{@migration_root}/rds"
+        instances_dir = "#{rds_dir}/instances"
+
+        if !Dir.exists?(@migration_root)
+          Dir.mkdir(@migration_root)
+        end
+        if !Dir.exists?(rds_dir)
+          Dir.mkdir(rds_dir)
+        end
+        if !Dir.exists?(instances_dir)
+          Dir.mkdir(instances_dir)
+        end
+
+        RDS.named_instances.each do |name, instance|
+          puts "Migrating #{name}..."
+
+          cumulus_instance = InstanceConfig.new(name).populate!(instance)
+
+          json = JSON.pretty_generate(cumulus_instance.to_hash)
+          File.open("#{instances_dir}/#{name}.json", "w") { |f| f.write(json) }
+        end
+      end
+
+      def update(local, diffs)
+        aws_instance = RDS::named_instances[local.name]
+        client = Aws::RDS::Client.new(Configuration.instance.client)
+
+        modify_instance = Proc.new do |options_hash|
```

Comment:
Is there any way to batch these up so we don't make a bunch of calls if we decide to change several attributes at once?